test/e2e/run.sh

```diff
@@ -64,8 +73,23 @@ do
   # Some of the tests will modify files in the distribution folder, e.g. cluster test will modify the application.yml
   # so we give each test a separate distribution folder here
   mkdir -p "$test_case" && tar -zxf dist/apache-skywalking-apm-bin.tar.gz -C "$test_case"
-
-  ./mvnw -Dbuild.id="${BUILD_ID:-local}" -Dsw.home="${base_dir}/$test_case/apache-skywalking-apm-bin" -f test/e2e/pom.xml -pl "$test_case" -am verify
+  
+  sky_home_dir="${base_dir}/$test_case/apache-skywalking-apm-bin"
+  
+  if ${use_mysql} -eq 1; then
+      echo "MySQL database is storage provider..."
+      
+      # Download MySQL connector.
+      curl ${MYSQL_URL} > "${sky_home_dir}/oap-libs/${MYSQL_DRIVER}"
+      [[ $? -ne 0 ]] && echo "Fail to download ${MYSQL_DRIVER}." && exit 1
+      
+      # Modify application.yml to set MySQL as storage provider.
+      cat "${sky_home_dir}/conf/application.yml" | sed '/elasticsearch/,/mysql/d' | sed "/storage:/a \  mysql:" | sed "/storage:/,/receiver-sharing-server:/s/#//" > ${TMP_APP_YML}
+      cat ${TMP_APP_YML} > "${sky_home_dir}/conf/application.yml"
+      rm -f ${TMP_APP_YML}
```

Comment:
These won't actually work, because **the `application.yml` is modified inside the docker container just before the OAP backend starts**, 

https://github.com/apache/skywalking/blob/bde6c137b70b10ceecc5c4ca53fecf99e263f57d/test/e2e/e2e-cluster/test-runner/src/docker/rc.d/rc0-prepare.sh#L23-L32

so the changes here will be overrode.

Reply:
Hi, thanks for your reply.
Before executing `awk -f /clusterize.awk application.yml`, application.yml has been modified by `run.sh`.  Will the changes from `clusterize.awk` and `run.sh` be kept togather in `clusterized_app.yml`?

Reply:
> Hi, thanks for your reply.
> Before executing `awk -f /clusterize.awk application.yml`, application.yml has been modified by `run.sh`. Will the changes from `clusterize.awk` and `run.sh` be kept togather in `clusterized_app.yml`?

Yes, I missed that, they will be kept, but you still need to follow this https://github.com/apache/skywalking/pull/3648#issuecomment-543495629